testing/custom_toolchain_auth/WORKSPACE

```diff
@@ -0,0 +1,38 @@
+workspace(name = "e2e_testing")
+
+# Placeholder repository definition to import rules_docker. To be replaced
+# with real path on bazel invocation using --override_repository
+local_repository(
+    name = "io_bazel_rules_docker",
+    path = "replace/this/path",
```

Comment:
cant you just do ../ here?

Reply:
Nope, when I do "../.." the path resolves to some bazel execroot path and I get

no such package '@io_bazel_rules_docker//toolchains/docker' /usr/local/google/home/suvanjan/.cache/bazel/_bazel_suvanjan/1e9e341f8f17444b215554700e5f65bc/external/io_bazel_rules_docker